cloud-sql.tf

```diff
@@ -0,0 +1,158 @@
+  combiner     = "OR"
+  # Available severity levels: INFO, WARNING, ERROR, CRITICAL, available from google provider >= 5.0
+  # severity     = each.value.severity
+
+  conditions {
+    condition_threshold {
+      filter          = "resource.type = \"cloudsql_database\" AND resource.labels.database_id = \"${local.cloud_sql_project}:${each.value.instance}\" AND metric.type = \"cloudsql.googleapis.com/database/cpu/utilization\""
+      comparison      = "COMPARISON_GT"
+      threshold_value = each.value.threshold
+      duration        = each.value.duration
+      trigger {
+        count = 1
+      }
+      aggregations {
+        alignment_period   = each.value.alignment_period
+        per_series_aligner = "ALIGN_MEAN"
+      }
+    }
+    display_name = "${local.cloud_sql_project} ${each.value.instance} - CPU utilization ${each.value.severity} ${each.value.threshold * 100}%"
+  }
+  alert_strategy {
+    auto_close = local.cloud_sql_auto_close
+  }
+  notification_channels = local.cloud_sql_notification_channels
+}
+
+# ----------------------
+# CloudSQL Memory utilization
+# ----------------------
+resource "google_monitoring_alert_policy" "cloud_sql_memory_utilization" {
+  for_each = local.cloud_sql_memory_utilization
+
+  display_name = "${local.cloud_sql_project} ${each.value.instance} - Memory utilization ${each.value.severity} ${each.value.threshold * 100}%"
+  combiner     = "OR"
+  # Available severity levels: INFO, WARNING, ERROR, CRITICAL, available from google provider >= 5.0
+  # severity     = each.value.severity
+  conditions {
+    display_name = "${local.cloud_sql_project} ${each.value.instance} - Memory utilization ${each.value.severity} ${each.value.threshold * 100}%"
+    condition_threshold {
+      filter          = "resource.type = \"cloudsql_database\" AND resource.labels.database_id = \"${local.cloud_sql_project}:${each.value.instance}\" AND metric.type = \"cloudsql.googleapis.com/database/memory/utilization\""
+      duration        = each.value.duration
+      comparison      = "COMPARISON_GT"
+      threshold_value = each.value.threshold
+
+      aggregations {
+        alignment_period   = each.value.alignment_period
+        per_series_aligner = "ALIGN_MEAN"
+      }
+    }
+  }
+
+  alert_strategy {
+    auto_close = local.cloud_sql_auto_close
+  }
+
+  notification_channels = local.cloud_sql_notification_channels
+}
+
+# ----------------------
+# CloudSQL disk utilization
+# ----------------------
+resource "google_monitoring_alert_policy" "cloud_sql_disk_utilization" {
+  for_each = local.cloud_sql_disk_utilization
+
+  display_name = "${local.cloud_sql_project} ${each.value.instance} - Disk utilization ${each.value.severity} ${each.value.threshold * 100}%"
+  combiner     = "OR"
+  # Available severity levels: INFO, WARNING, ERROR, CRITICAL, available from google provider >= 5.0
```

Comment:
```suggestion
  # @TODO: restore severity after all projects are migrated to google provider 5.x. Actually, we cannot set srverity since zambon-ops is using a 4.x version.
```